rawkv/rawkv.go

```diff
@@ -71,6 +71,13 @@ type Client struct {
 	regionCache *locate.RegionCache
 	pdClient    pd.Client
 	rpcClient   client.Client
+	atomic      bool
```

Comment:
What's this flag used for? why does it only affect `RawBatchGet` and `RawBatchPut` but not `CAS`?

Reply:
This `atomic` flag is supplied for raw requests related to `write` operations like `RawPut` and `RawDelete`. It won't affect to read operation like `RawGet`. In TiKV, the `CAS` operation is implemented by the mechanism of `latches` of the transaction module. While CAS using `latch` to write, the other write operations need to use this mode either, or It will break the linearizability of TiKV.

Reply:
Can we use RawCASRequest in RawBatchPut？

Reply:
@neicunlideyu For now We can't.